newrelic/resource_newrelic_alert_condition.go

```diff
@@ -118,7 +118,7 @@ func resourceNewRelicAlertCondition() *schema.Resource {
 			},
 			"condition_scope": {
 				Type:     schema.TypeString,
-				Optional: true,
+				Required: true,
```

Comment:
Is this a breaking change?

Reply:
The API apparently throws an error and this is required per our docs, so this should be required so the update is correct.